### ISSUE TITLE
[Feature request] War module - special rules for towns at war

### ISSUE THREAD

**Issue by farum12** [WiIIiam278/HuskTowns#378]
As a war participant,
I want to interact with enemy town, break blocks, etc.
But I don't want my town to be invaded by non-war participant
And my enemies don't want their town to be invaded by non-war participants

**Current behavior:**
Town Valhalla declares war on town Stalgard.
Both towns agree, pay wagers.
War commences.
Per wartime_rules, both towns are now public, making them open for attackers/defenders.
This also means that non-participants, eg. players from none of those towns are able to break blocks, use containers, grief, interact, etc.

Long story short - current implementation of War module gives ability for non-war participants to gain advantage of both towns, which are participating in war. There should be an option to turn wartime_rules ONLY for players which are participating in War.
